app/src/main/res/values/strings.xml

```diff
@@ -424,7 +424,9 @@ https://github.com/RocketChat/java-code-styles/blob/master/CODING_STYLE.md#strin
     <string name="save">SAVE</string>
     <string name="no_thanks">NO THANKS</string>
     <string name="chat_with">Chat with %1$s on %2$s</string>
-    <string name="default_invitation_text">Viasat Connect is a new application that makes it easy for me to text with friends and family. Download the application here %1$s and connect with me.</string>
+    <string name="default_invitation_text_description">Viasat Connect is a new application that makes it easy for me to text with friends and family.</string>
+    <string name="default_invitation_text_dynamic_link">Download the application here %1$s or get it by searching in the Google Play Store and connect with me.</string>
+    <string name="default_invitation_text_app_store">Download the application from the Google Play Store and connect with me.</string>
```

Comment:
suggested change: "Download the application by searching for Viasat Connect in the Google Play Store to chat with me."